cra-tf-validate-ignore-goals.json

```diff
@@ -0,0 +1,14 @@
+{
+    "scc_goals": [
+        {
+            "scc_goal_id": "3000408",
+            "description": "Check whether Flow Logs for VPC are enabled",
+            "is_valid": true
```

Comment:
this needs to be set to false. and then add another field:
`"ignore_reason": "This is part of the test example and not the current module",`